### PR TITLE
german translation 1-3

### PR DESCRIPTION
**Pull request by afink70**

### CODE REVIEW

File: de/1/math_operations.md [hunk 1]
```diff
@@ -1,6 +1,28 @@
...
 Do NOT change the code below. The below code runs the code editor -->
+# Chapter 3: Mathematische Operationen
 
+Mathematik in Vyper ist ziemlich unkompliziert. Die folgenden Operationen sind die gleichen wie in den meisten anderen Programmiersprachen:
+
+- Addition: `x + y`
+- Subtraktion: `x - y`
+- Multiplikation: `x * y`
+- Division: `x / y`
+- Divison mit Rest: `x % y` (Zum Beispiel `13 % 5` ist `3`, weil beim Dividieren von `13` durch `5`, bleibt der Rest `3`)
```
Comment (vasa-develop) [inline de/1/math_operations.md]: Update from `Divison mit Rest` to `Modul / Rest`

**Review by vasa-develop — CHANGES_REQUESTED** [review]
This one is almost ready for merge. Just make the change as suggested :)

File: de/1/math_operations.md (hunk 1 above)
Comment (finki70) [reply de/1/math_operations.md:11]: Well, I think the original term should be better. https://de.wikipedia.org/wiki/Division_mit_Rest

Comment (afink70) [reply de/1/math_operations.md]: In my oppinion, we should use the original term 

https://de.wikipedia.org/wiki/Division_mit_Rest

**Pull request merged**